gef.py

```diff
@@ -6248,9 +6248,9 @@ def invoke(self, args, from_tty):
 def __gef_prompt__(current_prompt):
     prompt = "gef> " if PYTHON_MAJOR == 2 else "gef\u27a4  "
     if is_alive():
-        fmt_prompt = "\001\033[1;32m\002{0:s}\001\033[0m\002".format(prompt)
+        fmt_prompt = "\033[1;32m{0:s}\033[0m".format(prompt)
     else:
-        fmt_prompt = "\001\033[1;31m\002{0:s}\001\033[0m\002".format(prompt)
+        fmt_prompt = "\033[1;31m{0:s}\033[0m".format(prompt)
```

Comment:
Those sequences are interpreted by readline, so it shouldn't matter whether `\x01` is used or `\001`. Both crash the debugger.